desktop/hardened-desktop/index.md

```diff
@@ -0,0 +1,55 @@
+---
+title: Hardened Desktop
+description: Overview of what Hardened Desktop is
+keywords: security, hardened desktop, enhanced container isolation, registry access management, admin controls, root access, administrators, docker desktop
+---
+>Note
+>
+>Hardened Desktop is available to Docker Business customers only.
+
+Hardened Desktop is a new security model for Docker Desktop. It's part of Docker's ongoing effort to increase Docker Desktop security without impacting the developer experience.
+
+The Hardened Desktop security model provides administrators with a simple and powerful way to increase the security of their containerised development and moves the ownership boundary for containers to the organization, meaning that any security controls administrators set cannot be altered by the user.
+
+It is designed specifically for organizations who don’t give developers root access to their machines, and who would like Docker Desktop to be within the organization’s centralized control.
+
+Hardened Desktop includes:
+- Admin Controls, which helps administrators to confidently manage and control usage of Docker Desktop.
+- Enhanced Container Isolation, an admin setting that instantly enhances security by preventing containers from running as root in Docker Desktop’s Linux VM.
```

Comment:
can we extend this to also mention that it ensures admin controls cannot be modified by containers ?

Reply:
Also Enhanced Container Isolation is not an admin-only setting; it can be set by developers (currently via the settings.json file, in the future probably through the UI). Admins can however configure the default value and lock the setting via the admin-settings.json file, just like other settings. (FYI @christophermclellan).